src/web/components/ApiKeyManagement/KeyTable.tsx

```diff
@@ -32,12 +32,14 @@ function KeyTable({ apiKeys, onKeyEdit, onKeyDisable, availableRoles }: KeyTable
         <thead>
           <tr>
             <th>Name</th>
-            <th className='key-id-header'>
-              <div>Key ID</div>
-              <Tooltip>
-                Shows the first few characters of your plain text key. If support related to a key
-                is required, please provide this ID.
-              </Tooltip>
+            <th>
+              <div className='tooltip-header'>
```

Comment:
hm `key-id-header` sorta seems like a better name to me. Any particular reason to name the className here?

Reply:
Yeah that makes sense. Sorry, had another change I reverted back and left that behind.